control_software/scripts/hera_snap_redis_monitor.py

```diff
@@ -275,16 +275,57 @@ def print_ant_log_messages(corr):
         corr.r.hmset('version:%s:%s' % (__package__, os.path.basename(__file__)), {'version':__version__, 'timestamp':datetime.datetime.now().isoformat()})
 
         # Get antenna stats
-        input_stats = corr.do_for_all_f("get_stats", block="input", kwargs={"sum_cores": True})
+        # do_for_all_f breaking on adc stats, casting to None 25 Aug 2020
+        # input_stats = corr.do_for_all_f("get_stats", block="input", kwargs={"sum_cores": True})
```

Comment:
Add comment here that removing adc histogram circuit in latest bitfile necessitates using adc snapshots to build histograms in software.